### DISCUSSION TITLE
How to Update?

### DISCUSSION THREAD

**Discussion by ericdrgn** [upptime/upptime#327] [Q&A] [answered]
~~So I feel pretty stupid and confused... But how do I keep my upptime status page up to date with changes? I created from the template but can see a few updates have come through and mine isn't updated (I guess I thought it would happen automatically) I'm on 1.26.1 and can't seem to figure out how to get it updated. Help?~~

Nevermind. I was being stupid, all good now!

**Comment by gitgrub** [comment]
Same here, I am on 1.26.1 (the version I startet with) but 1.26.3 is out sice two days.
I thought source update would happen on a daily base.
Updates CI was run successfully, green state.

**Comment by AnandChowdhary** [comment, marked as answer]
This fix has been released, but if it's not in your repository, please do this manually:

1. Find and replace `GITHUB_TOKEN` with `GH_PAT` in your `.github/workflows/update-template.yml`
2. Run Update Template CI [manually](https://upptime.js.org/docs/triggers)
3. This should auto-update your Upptime to the latest version, which both fixes the updating problem and now throws an error in case the update couldn't be pushed.

**Reply by gitgrub** [reply]
Thank you, this worked and I am up to date :)

**Reply by ericdrgn** [reply]
Sorry @AnandChowdhary, I don't imagine myself to be stupid but I feel stupid haha. I applied your changes and run the CI manually and it doesn't look like anything updated unless I am just supposed to wait for it to do something now.

**Reply by AnandChowdhary** [reply]
Seems like you're running the latest version now: https://github.com/ericdrgn/drgnstatus/commit/a369883c98e95f1f0b76653324384718fa2b016a.

**Comment by fraserbooth** [comment]
I've noticed that mine is also not updating, so made the GH_PAT change suggested above, but the workflow is still failing.

Does the following error message mean anything to anyone?

Fetching the repository
  /usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +13d108865b96f1f1a07527ac3a24[43](https://github.com/fraserbooth/fb-monitor/actions/runs/3714775335/jobs/6302656555#step:2:48)1ee5159f08:refs/remotes/origin/master
  Error: fatal: could not read Username for 'https://github.com/': terminal prompts disabled
  The process '/usr/bin/git' failed with exit code 128
  
  
 PS: I love this Upptime Monitor, and have been recommending it far and wide :)

**Reply by AnandChowdhary** [reply]
Would it be possible for you to share your repository or all the logs? I'd be curious to figure out what's up!

**Reply by fraserbooth** [reply]
That's kind thank you :)

The Repo is at:
https://github.com/fraserbooth/fb-monitor

...not sure what access to the logs is given automatically on public repos, but I've invited you as a collaborator if that helps?

**Reply by AnandChowdhary** [reply]
And can you confirm that you indeed have the repository secret GH_PAT set up?

**Reply by fraserbooth** [reply]
Sorry for the delayed response. Busy last days before the Christmas Holiday!

Yes, I set up GH_PAT when I first started this monitor. It says "Updated on Apr 10, 2021", however for some reason it doesn't appear in my Personal Access Tokens list anymore. I'll try adding a new one and updating.

**Reply by fraserbooth** [reply]
That seems to have fixed it. I wonder why that PAT got automatically deleted.

Thank you for your help, and sorry to have wasted your time :)

**Reply by AnandChowdhary** [reply]
No worries, glad to know its works! :)
